index.html

```diff
@@ -14,6 +14,17 @@
   <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/main.css">
 
+  <link rel="stylesheet" type="text/css" href="./css/addtohomescreen.css">
+  <script src="./js/addtohomescreen.min.js"></script>
+  <script>
+      addToHomescreen();
```

Comment:
Use as little script tags in HTML as possible, include this in main.js if we end up using it?